src/plugins/plugin.ts

```diff
@@ -11,7 +11,9 @@ export interface PluginConfig {
     pluginName: string
     identifier: string
     successDecryptionInspector?: PluginInjectFunction<{ message: TypedMessage }>
+    pageInspector?: React.ComponentType<{}>
     postInspector?: PluginInjectFunction<{}>
+    postMessageProcessor?: (message: TypedMessageCompound) => TypedMessageCompound
```

Comment:
Does the post here means "after" like "post-install"? If so, please rename it cause it brings confusion with (user) post